command/sub8_missions/missions/align_channel.py

```diff
@@ -1,49 +1,72 @@
 from txros import util
+import rospy
+import tf
 from sub8_ros_tools import normalize, clip_norm
+from image_geometry import PinholeCameraModel
 import numpy as np
 
 
 @util.cancellableInlineCallbacks
-def get_above(sub):
-    '''TODO:
-    Use knowledge of height from DVL
+def run(sub_singleton):
+    # Right now we assume the marker is in our field of view. (Will add searching later)
+    # Go to max height to see as much as we can.
+
+    rospy.init_node('align_channel_mission')
+    tf_listener = tf.TransformListener()
```

Comment:
Later, use the sub object's tf listener so that you don't have to wait for a buffer